security/xca/PortFile

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem                  1.0
+
+name                        xca
+version                     2.1.2
+
+categories                  security crypto
+platforms                   darwin
+license                     xca
+
+maintainers                 {@lhaeger} openmaintainer
+
+description                 X Certificate and Key management
+long_description            XCA is a CA & PKI management tool that supports creating and managing X.509 certificates, \
+                            certificate requests, RSA, DSA and EC private keys, Smartcards and CRLs.
+homepage                    https://www.hohnstaedt.de/xca/
+
+PortGroup                   github 1.0
+
+github.setup                chris2511 ${name} ${version} RELEASE.
+github.tarball_from         releases
```

Comment:
Please move these 3 lines (`PortGroup`, `github.setup` and `github.tarball_from`) to the top of Portfile, just below the `PortSystem` line. Then the name and version are directly added and set in the `github.setup` line. Then the current `name` and `version` lines are redundant and can be removed.